business/sys/database/database.go

```diff
@@ -264,9 +263,9 @@ func NamedQueryStruct(ctx context.Context, log *zap.SugaredLogger, db sqlx.ExtCo
 	return nil
 }
 
-// queryString provides a pretty print version of the query and parameters.
-func queryString(query string, args ...any) string {
-	query, params, err := sqlx.Named(query, args)
+// queryString provides a pretty print version of the query and parameter.
+func queryString(query string, arg any) string {
```

Comment:
I didn't see this PR or I would have accepted it. I already made this change.  Thanks!!!